base/server/src/main/java/com/netscape/cmscore/security/PWsdrCache.java

```diff
@@ -284,12 +284,12 @@ public void deleteEntry(String tag) throws EBaseException {
             //                if tag doesn't exist
             Hashtable<String, String> ht = string2Hashtable(dcrypts);
 
-            if (ht.containsKey(tag) == false) {
-                logger.debug("PWsdrCache: tag: " + tag + " does not exist");
-                return;
-            } else {
+            if (ht.containsKey(tag)) {
                 logger.debug("PWsdrCache: deleting tag: " + tag);
                 ht.remove(tag);
+            } else {
+                logger.debug("PWsdrCache: tag: " + tag + " does not exist");
+                return;
```

Comment:
Should we keep the original if-block since it returns and unindent the else-block instead?